src/go/rpk/pkg/cli/cmd/topic/create.go

```diff
@@ -24,13 +24,27 @@ import (
 	"github.com/twmb/franz-go/pkg/kmsg"
 )
 
+type topic struct {
+	Name    string `json:"name" yaml:"name"`
+	Message string `json:"message" yaml:"message"`
+	Error   bool   `json:"error" yaml:"error"`
+}
+type topicsCollection struct {
+	Topics []topic `json:"topics" yaml:"topics"`
+}
+
+func (collection *topicsCollection) AddTopic(newTopic topic) {
```

Comment:
I think I'd prefer an append at the callsite rather than AddTopic here -- this feels a bit like an abstraction that isn't quite necessary